ang/civicase/ContactCaseTab.html

```diff
@@ -11,17 +12,105 @@ <h1 crm-page-title class="element-invisible">Related cases to the user</h1>  <!-
     <div class="clearfix" ng-if="totalCount > 0">
       <div class="civicase__contact-cases-tab-container pull-left col-md-6">
         <div  
-          ng-repeat="case in casesListConfig" 
+          ng-repeat="casesList in casesListConfig"
           class="panel panel-default civicase__contact-cases-tab-list">
           <div class="panel-heading">
-            <h3 class="panel-title"> {{ case.title }}</h3>
+            <h3 class="panel-title"> {{ casesList.title }}</h3>
           </div>
-          <div class="panel-body">
-            <civicase-contact-case-tab-case-list case-obj="case"></civicase-contact-case-tab-case-list>
+          <div class="panel-body panel-body-extra">
+            <civicase-contact-case-tab-case-list cases-list="casesList" viewing-case-id="selectedCase.id"></civicase-contact-case-tab-case-list>
           </div>
         </div>
       </div>
-    </div>      
+      <div class="panel panel-default civicase__contact-cases-panel civicase__contact-cases-container pull-left col-md-6">
```

Comment:
`civicase__contact-cases-container` - to contains the left section and right section
`civicase__contact-cases-panel` - to add panel specific styles to the contact-cases-tab

However, this is right that it shouldn't be on the same div so fixed the HTML so that it looks like

<img width="1309" alt="c51-152 - using the html correctly" src="https://user-images.githubusercontent.com/3340537/47482097-32be0800-d853-11e8-8482-4aab4ec1fd3d.png">


*PS*: Class name has been updated to `civicase__contact-cases-tab-container` and `civicase__contact-cases-tab__panel` respectively.